embark/lib/templates/README.md.eex

```diff
@@ -1,41 +1,64 @@
 # <%= app_module %>
 
-### Start services
+## Set Environment Variables
+
+Open the `hello_tokumei/.env` file to change environment variables to match your needs.
```

Comment:
There is a variable `<%= app_name %>` which can be used instead of just the example hello_tokumei

Reply:
I will use it.